providers/evercrypt/fst/EverCrypt.Hash.fst

```diff
@@ -549,7 +692,7 @@ let copy #a s_src s_dst =
       let s_dst: state SHA2_512 = s_dst in
       let p_dst = SHA2_512_s?.p !*s_dst in
       B.blit p_src 0ul p_dst 0ul 8ul
-  | Blake2S_s p_src ->
+  (*| Blake2S_s p_src ->
```

Comment:
this is the part that I need to fix -- @karthikbhargavan I think you said there was no hope of copying a blake2b/32 state into a blake2b/256 state?